src/tests/auth/resetpassword.test.js

```diff
@@ -0,0 +1,101 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../server';
+import { forgetPassword, createUser, userToForgetPassword, forgotPasswordToken, tokenToVerify, passwordMatch, passwordDonotMatch, validate } from '../fixtures/user.fixture';
+
+chai.should();
+chai.use(chaiHttp);
+
+const userVerificationTest = () => {
+	describe('/POST forget password', () => {
+		before(() => {
+			createUser(forgetPassword);
+		});
+		it('Should help a user who forgot the password', (done) => {
+			chai.request(app)
+				.post('/api/auth/forget-password')
+				.set('Authorization', `Bearer ${forgotPasswordToken}`)
```

Comment:
why are you still sitting authorization?

Reply:
done